builder/builder.go

```diff
@@ -471,7 +471,7 @@ func (b *Builder) saveArtifactLocally(ctx context.Context, artifact domain.Artif
 		}
 		return fmt.Errorf("cannot save artifact %s, since it does not exist", artifact.StringCanonical())
 	}
-	isWildcard := (len(fromGlobMatches) > 1)
+	isWildcard := strings.ContainsAny(fromPattern, `*?[`)
```

Comment:
Are we shooting for compatibility with `filepath.Match`? https://golang.org/pkg/path/filepath/#Match

Reply:
Yes.. it looks like this is the kind of check filepath uses underneath.